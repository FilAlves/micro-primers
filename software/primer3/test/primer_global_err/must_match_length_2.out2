../src/primer3_core: PRIMER_MUST_MATCH_THREE_PRIME must have 5 characters
