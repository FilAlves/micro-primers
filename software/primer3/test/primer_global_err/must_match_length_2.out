SEQUENCE_ID=must_match_length_2
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_MUST_MATCH_THREE_PRIME=NNNN
PRIMER_ERROR=PRIMER_MUST_MATCH_THREE_PRIME must have 5 characters
=
